release_util_script__deploy_to_maven_central.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -euo pipefail
+#------------------------------------------------------------
+VERSION=$(mvn help:evaluate -Dexpression=project.version | grep -v -e "^\\[")
+RELEASE="true"
+PGP_KEY_ENC_FILENAME="private.gpg.enc"
```

Comment:
Should this variable actually be `GPG_KEY_ENC_FILENAME`?

Reply:
GnuPG as an implementation of pgp. I guess pgp sticks in my head better (pretty good privacy...) - so I used this on the var name. 

But inconsistent. Changed.